modules/yaml-cpp/0.8.0/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/stonier/yaml-cpp/releases/download/0.8.0/yaml-cpp-0.8.0.tar.gz",
```

Comment:
Would it be possible to use the upstream repo here and apply patches?

Reply:
Ah, I just added information in the description pertaining to that :)

Reply:
Thanks, that makes sense. Maybe we could wait for a few days? Maybe we then don't have to choose between an archive from a fork and an unstable source.

Reply:
> Thanks, that makes sense. Maybe we could wait for a few days?

Yes, I'm fine with that.

Reply:
Did this work out? If not, we can merge as is.

Reply:
Yes and no. A PR for the bazel module patches got in, and a PR to upload a release asset on future releases, but he hasn't gotten around to uploading any release assets for current or prior releases, so this one might have to do for now.